ci/setup_env.sh

```diff
@@ -36,9 +36,13 @@ else
   exit 1
 fi
 
-wget -q "https://repo.continuum.io/miniconda/Miniconda3-latest-$CONDA_OS.sh" -O miniconda.sh
+if [ "${TRAVIS_CPU_ARCH}" == "arm64" ]; then
+  wget -q "https://github.com/conda-forge/miniforge/releases/download/4.8.2-1/Miniforge3-4.8.2-1-Linux-aarch64.sh" -O miniconda.sh;
+else
+  wget -q "https://repo.continuum.io/miniconda/Miniconda3-latest-$CONDA_OS.sh" -O miniconda.sh;
```

Comment:
I'd prefer not to repeat the `wget`. You can add something like:
```bash
CONDA_URL="https://repo.continuum.io/miniconda/Miniconda3-latest-$CONDA_OS.sh"
if [ "${TRAVIS_CPU_ARCH}" == "arm64" ]; then
    CONDA_URL="https://github.com/conda-forge/miniforge/releases/download/4.8.2-1/Miniforge3-4.8.2-1-Linux-aarch64.sh"
fi

wget -q $CONDA_URL -O miniconda.sh;
```

Reply:
@datapythonista , 

Thanks for the suggestion. I will update as suggested and add a commit.